backend/app/services/prompt_draft/fallback.py

```
@@ -0,0 +1,198 @@
+        return True
+    if normalized.isdigit():
+        return True
+    return False
+
+
+def _infer_assistant_identity(user_lines: list[str]) -> tuple[str, str]:
+    joined = " ".join(user_lines)
+    if any(keyword in joined for keyword in ("流程图", "mermaid", "架构图", "时序图")):
+        return (
+            "流程图协作助手",
+            "帮助用户梳理流程、补齐关键信息，并输出可执行的流程图方案",
```

Ruff (0.15.7)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)